Shared/mods/deathmatch/logic/lua/LuaBasic.h

```diff
@@ -68,6 +68,26 @@ namespace lua
         return 1;
     }
 
+    inline int Push(lua_State* L, const CLuaArgument* const arg)
```

Comment:
Do we actually need this overload? In most cases the reference version is likely the correct one to use, especially since `nullptr` is not allowed.